index.yaml

```diff
@@ -121,8 +121,6 @@ system:
     reference: ./docker
   tasks-workflow:
     reference: ./tasks-workflow
-  safe-search:
```

Comment:
I do have to remove this reference since darren remove the gpt file. 

https://acorn-io.slack.com/archives/C03FU91U4SX/p1742109501278959